changelog.d/14978.bugfix

```diff
@@ -0,0 +1 @@
+Add the ability to enable/disable registrations when in the OIDC flow.
```

Comment:
This sounds more like a feature than a bugfix to me?

Reply:
renamed 👍